graph/pull_v2.go

```diff
@@ -237,7 +237,7 @@ func (p *v2Puller) pullV2Tag(tag, taggedName string) (verified bool, err error)
 
 	out.Write(p.sf.FormatStatus(tag, "Pulling from %s", p.repo.Name()))
 
-	downloads := make([]downloadInfo, len(manifest.FSLayers))
+	var downloads []*downloadInfo
```

Comment:
I think it wouldn't hurt here to use nil slice for consistency. Performance gain on slice reallocating in our case is close to nothing.